arm-network/compositeNetworkClient.json

```diff
@@ -0,0 +1,21 @@
+{
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "Composite Swagger for Network Client"
+  },
+  "documents": [
+    "./2016-09-01/swagger/applicationGateway.json",
+    "./2016-09-01/swagger/network.json",
+    "./2016-09-01/swagger/routeTable.json",
+    "./2016-09-01/swagger/publicIpAddress.json",
+    "./2016-09-01/swagger/networkSecurityGroup.json",
+    "./2016-09-01/swagger/loadBalancer.json",
+    "./2016-09-01/swagger/virtualNetwork.json",
+    "./2016-09-01/swagger/networkInterface.json",
+    "./2016-09-01/swagger/vmssNetworkInterface.json",
+    "./2016-09-01/swagger/usage.json",
+    "./2016-09-01/swagger/checkDnsAvailability.json",
+    "./2016-09-01/swagger/virtualNetworkGateway.json",
+    "./2016-09-01/swagger/expressRouteCircuit.json",
```

Comment:
@DeepakRajendranMsft - Can you remove the extra comma from the last line in the documents array?